CHANGELOG.rst

```diff
@@ -1,3 +1,76 @@
+Sunpy 2.0.2 (2020-09-26)
+========================
+
+Deprecations and Removals
+-------------------------
+
+- Removed deprecated functions:
+
+  `sunpy.coordinates.frames.Helioprojective.calculate_distance`, alternative is `sunpy.coordinates.frames.Helioprojective.calculate_distance.make_3d`.
+  ``sunpy.image.coalignment.repair_image_nonfinite`` - if you wish to repair the image, this has to be done manually before calling the various `sunpy.image.coalignment` functions. The ``repair_nonfinite`` keyword argument to ``calculate_shift`` and  ``calculate_match_template_shift`` has been removed.
+  `sunpy.instr.lyra.download_lytaf_database` - this just downloaded the file at ``http://proba2.oma.be/lyra/data/lytaf/annotation_ppt.db``, which can be done manually.
+  `sunpy.util.net.check_download_file`, no alternative.
+  `sunpy.visualization.animator.ImageAnimatorWCS`, alternative is `sunpy.visualization.animator.ArrayAnimatorWCS`. (`#4350 <https://github.com/sunpy/sunpy/pull/4350>`__)
+
+
+Bug Fixes
+---------
+
+- Fixed a bug in `~sunpy.net.dataretriever.sources.goes.XRSClient._get_url_for_timerange` which returned incorrect URLs
+  because of not using ``**kwargs`` in the client's ``_get_overlap_urls()`` method. (`#4288 <https://github.com/sunpy/sunpy/pull/4288>`__)
+- Data products from `~sunpy.net.dataretriever.NOAAIndicesClient` and
+  `~sunpy.net.dataretriever.NOAAPredictClient` have been updated to download
+  new JSON files. The old text files which the data used to come in no longer
+  exist. The new JSON files for `~sunpy.net.dataretriever.NOAAIndicesClient`
+  now do not have the following columns:
+  - Geomagnetic Observed and Smoothed
+  - Sunspot Numbers Ratio (RI/SW)
+
+  Both `sunpy.timseries.sources.NOAAIndicesTimeSeries` and
+  `sunpy.timseries.sources.NOAAPredictIndicesTimeSeries` have been updated to
+  support the new JSON files. Loading the old text files is still supported,
+  but support for this will be removed in a future version of sunpy. (`#4340 <https://github.com/sunpy/sunpy/pull/4340>`__)
```

Comment:
I know this isn' t really relevant here but if we have deprecated this then it should be in the correct section. Also actually relevant here is we shouldn't backport deprecation warnings.